src/user/supla_esp_devconn.c

```diff
@@ -37,6 +37,11 @@
 #include "supla-dev/srpc.h"
 #include "supla-dev/log.h"
 
+#if defined(POWSENSOR2)
```

Comment:
It does not change anything in the abcd file. It is not necessary to implement this functionality.